src/main/java/com/okta/sdk/framework/ApiClient.java

```diff
@@ -39,6 +40,7 @@
     protected String token;
     protected ApiClientConfiguration configuration;
     private final Logger LOGGER = LoggerFactory.getLogger(ApiClient.class);
+    private HttpResponse lastHttpResponse;
```

Comment:
This should probably be an `AtomicReference<HttpResponse>` rather than just an HttpResponse. This way, multiple threads can make calls using this ApiClient and each one is guaranteed to see the latest value for `lastHttpResponse` (because to access it you will have to do `httpResponseRef.get()`).